tests/e2e/release-e2e-scenarios.md

```
@@ -0,0 +1,638 @@
+{"custom_id":"qa-batch-1","method":"POST","url":"/v1/chat/completions","body":{"model":"gpt-4.1-nano","messages":[{"role":"user","content":"Reply with exactly QA_BATCH_FILE_OK"}],"max_tokens":20}}
+EOF
+
+printf 'qa file payload\n' > /tmp/qa-upload.txt
+
+export BATCH_FILE=/tmp/qa-openai-batch.jsonl
+export UPLOAD_FILE=/tmp/qa-upload.txt
+```
+
+## 1. Infra, discovery, observability
+
+### S01 Health endpoint
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)